sys-cluster/genders/genders-1.27.3.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_USE_SETUPTOOLS=no
+
+inherit autotools distutils-r1 perl-module java-pkg-opt-2
+
+case "${PV}" in
+9999)
+	SRC_URI=""
+	EGIT_REPO_URI="https://github.com/chaos/${PN}.git"
+	inherit git-r3
+	KEYWORDS=""
+	;;
+*)
+	MY_PV="$(ver_rs 1-2 -)"
+	MY_P="${PN}-${MY_PV}"
+	SRC_URI="https://github.com/chaos/${PN}/archive/${MY_P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}-${MY_P}"
+	;;
+esac
+
+DESCRIPTION="Genders is a static cluster configuration database used for cluster configuration management."
+HOMEPAGE="https://github.com/chaos/genders"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="cxx java perl python"
+
+CDEPEND="
+	perl?	( dev-lang/perl:= )
+	python?	( ${PYTHON_DEPS} )
+"
+DEPEND="
+	${CDEPEND}
+	java?	( virtual/jdk:1.8 )
+
+"
+
+RDEPEND="
+	${DEPEND}
+	java?	( virtual/jre:1.8 )
+"
+REQUIRED_USE="
+		python? ( ${PYTHON_REQUIRED_USE} )
+"
+
+src_prepare() {
+	sed -i "s|perl python||" src/extensions/Makefile.am
+	sed -i "s|\$(DESTDIR)\$(docdir)-\$(VERSION)-javadoc|\$(DESTDIR)\$(docdir)/html/javadoc|" src/extensions/java/Makefile.am
+	eapply_user
+	./autogen.sh
+}
+
+src_configure() {
+	use java && append-cflags "-I${S}/src/libgenders"
+	use java && append-cflags "$(java-pkg_get-jni-cflags)"
+
+	local myconf="(
+		--disable-static
+		--with-non-shortened-hostnames
+		$(use_with cxx cplusplus-extensions)
+		$(use_with java java-extensions)
+		$(use_with perl perl-extensions)
+		$(use_with python python-extensions)
+	)"
+	econf "${myconf[@]}"
+}
+
+src_compile() {
+	default
+
+	if use perl ; then
+		cd "${S}/src/extensions/perl"
+		perl-module_src_configure
+		perl-module_src_compile
+	fi
+
+	if use python; then
+		cd "${S}/src/extensions/python"
+		cp genderssetup.py setup.py
+		distutils-r1_src_compile
+	fi
+}
+
+src_install() {
+	default
+	dodoc README TUTORIAL NEWS
```

Comment:
Since you call `default` here that invokes `einstalldocs`, you'd better be putting these to global `DOCS` array.